### DISCUSSION TITLE
Fira Code: a fonte perfeita pra você codar

### DISCUSSION THREAD

**Discussion by leonardoelias** [frontendbr/forum#304] [Dicas]
Galera comecei hoje a usar a Fire Code, e estou gostando bastante : )

Link post : https://codigo.joselito.space/fira-code-c4337bdef1de#.9hstssad9
Link github: https://github.com/tonsky/FiraCode/tree/master/showcases

**Comment by paulinhoerry** [comment]
Eu gostava muito da Fira Code, até comprar a [_operator mono_](http://www.typography.com/fonts/operator/overview/).

**Comment by wmartins** [comment]
Uso a Fira Mono, mas gostaria de poder usar a Fira Code (acho que não funciona no meu terminal :( )

**Comment by adeonir** [comment]
Achei bacana a Operator, mas prefiro a Fira Code pelas ligaduras (que ainda não funcionam no Sublime Text, mas são show no Atom e o no iTerm).

@wmartins dá uma olhada no [suporte da fonte](https://github.com/tonsky/FiraCode#terminal-support).

**Comment by wmartins** [comment]
@adeonir, aí que tá, uso o xfce4-terminal no trabalho e o gnome-terminal em casa hahahaha :(

**Comment by leonardoelias** [comment]
@adeonir sim, fiquei muito de cara que não tem suporte ao sublime 👎  estou usando o Atom tbm.

**Comment by ghost** [comment]
To usando a Fira Mono e to gostando bastante, já estava enjoando da DejaVu Mono :yum:

**Comment by giioohbernini** [comment]
Nunca usei essas fontes diferentonas ai, mas achei legal. Atualmente to usando sublime, alguém me recomenda alguma? :D

**Comment by ghost** [comment]
@giioohbernini no sublime eu usava a DejaVu Sans Mono, é legalzinha :+1:

**Comment by expalmer** [comment]
@giioohbernini [Input Mono](http://input.fontbureau.com/preview/?size=14&language=python&theme=solarized-dark&family=InputSans&width=300&weight=300&line-height=1.2&a=0&g=0&i=0&l=0&zero=0&asterisk=0&braces=0&preset=default&customize=please)

**Comment by fdaciuk** [comment]
Eu gosto bastante da Consolas :heart: 
Microsoft só não faz SO decente.. de resto eles mandam bem :grin: :joy: :joy:

**Comment by napalmdev** [comment]
Funfa no VSCode  :wink:

**Comment by filipelinhares** [comment]
Eu usava a Fira Code mas conheci a _[Mononoki](http://madmalik.github.io/mononoki/)_.

**Comment by brunodevel** [comment]
Gosto bastante da [Source Code Pro](https://github.com/adobe-fonts/source-code-pro).

**Comment by davidwelberrs** [comment]
Monoid Nerd Font! Tem Ligatures e Icons Dev

**Comment by kvnol** [comment]
Tô testando a Fira e tô gostando.

**Comment by felipefialho** [comment]
Vou testar o Fira Code e o Mononoki, depois falo o que achei.

**Comment by adeonir** [comment]
No VSCode, precisa colocar `"editor.fontLigatures": true,` para usar com a Fira Code.

**Comment by felipefialho** [comment]
Mano, to usando e achando muito foda.

Fiz um [tweetzinho mostrando](https://twitter.com/felipefialho_/status/1134069145736273920)

**Comment by caioincau** [comment]
Eu usava mto também, até ir para a Operator Mono.

**Comment by brunooomelo** [comment]
`Consolas` ainda reina ❤️
